app/controllers/spree/api/favorites_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Spree
+  module Api
+    class FavoritesController < Spree::Api::BaseController
+
+      def index
+        @favorites = @current_api_user.favorites
+                                      .page(params[:page])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.